packages/pigeon/platform_tests/test_plugin/windows/test/null_fields_test.cpp

```diff
@@ -15,43 +15,26 @@ using flutter::EncodableList;
 using flutter::EncodableMap;
 using flutter::EncodableValue;
 
-// EXPECTs that 'map' contains 'key', and then returns a pointer to its value.
```

Comment:
nit: docstrings should have `///` prefix

Reply:
This is deleted code